cli/main.go

```diff
@@ -38,10 +43,16 @@ Support: support@dispatch.run
 		Title: "Dispatch Commands:",
 	})
 
+	// Account Management Commands
 	cmd.AddCommand(loginCommand())
 	cmd.AddCommand(switchCommand())
-	cmd.AddCommand(runCommand())
 	cmd.AddCommand(verificationCommand())
+
+	// Dispatch Commands
+	cmd.AddCommand(runCommand())
+
+	// Additional Commands
 	cmd.AddCommand(versionCommand())
+
```

Comment:
What do you think about removing the comments here? We don't have that many for now and it's pretty straightforward what they do, it'll keep the code concise.